README.md

```diff
@@ -42,6 +42,8 @@ Documentation
 -------------
 All documentation can be found within [the docs folder](./docs/).
 
+_**Note:** Links marked with an asterisk (__*__) use the third-party service [http://htmlpreview.github.io](http://htmlpreview.github.io) to render HTML._
```

Comment:
Not sure if this is a good idea, please let me know if you are against it and I can remove these

Reply:
I like that.